src/optimize.js

```diff
@@ -9,8 +9,12 @@ export default function(options) {
           warnings: false, // Because uglify reports irrelevant warnings.
         },
         sourceMap: false,
+        ie8: true,
+        safari10: true,
       },
       cache: true,
       parallel: true,
     })]
 }
+
+_.reduce(props, (res,key, value) => ({...res, key: value(props)}, {}))
```

Comment:
What's this piece of code doing here?

Reply:
thanks. it should not be there. My inattention in clearing.